index.html

```diff
@@ -95,8 +95,76 @@ <h1 class="introduction">
+              <div class="project-image-container">
+                <div id="mobile-close-button" class="mobile-close-button close-btn">
+                  <img src="./assets/images/cancel-white.svg" alt="close-icon" />
+                </div>
+                <div class="project-image">
+                  <img src="./assets/images/project-image.png" alt="project-image" />
+                </div>
+                <div class="mobile-project-image">
+                  <img src="./assets/images/mobile-project-image.png" alt="project-image" />
+                </div>
+              </div>
+              <div class="modal-head">
+                <h3>Keeping track of hundreds of components</h3>
+                <ul>
+                  <li>
+                    <a href="#" target="_blank"
+                      > <span>See Live</span> <img class="outlinks" src="./assets/images/live.svg" alt="project-image"
+                    /></a>
+                  </li>
+                  <li>
+                    <a href="#" target="_blank"
+                      ><span>See Source</span> <img class="outlinks" src="./assets/images/github-white.svg" alt="project-image"
+                    /></a>
+                  </li>
+                </ul>
+              </div>
+              <div class="mobile-skillset">
+                <p>Ruby on rails</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+              </div>
+              <div class="skillset">
+                <p>Codekit</p>
+                <p>Github</p>
+                <p>JavaScript</p>
+                <p>Bootstrap</p>
+                <p>Terminal</p>
+                <p>Codepen</p>
+              </div>
+              <div class="modal-description">
+                <p>
+                  Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur inventore
+                  sit itaque nihil accusamus dicta, perspiciatis aperiam mollitia libero dolores
+                  molestiae iste quo facilis rem repellendus adipisci aut provident laboriosam?
+                  Corrupti, perferendis alias animi officia ut inventore dignissimos, fugiat nostrum
+                  adipisci pariatur nam eaque a assumenda temporibus voluptate amet voluptates!
+                  Laboriosam fuga neque molestias quam et impedit mollitia quod maiores
+                </p>
+              </div>
+              <div class="modal-links">
+                <ul>
+                  <li>
+                    <a href="#" target="_blank">See Live
+                      <img class="outlinks" src="./assets/images/outlink.png" alt="project-image" /></a>
+                  </li>
+                  <li>
+                    <a href="#" target="_blank">See Source
+                      <img class="outlinks" src="./assets/images/github-white.svg" alt="project-image" /></a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
           <div class="card">
```

Comment:
- [ ] I kindly suggest that you add your projects and modals dynamically using javascript. 
- [ ] Kindly implement an array of objects to store your JavaScript objects containing the information for each project.

## EXAMPLE
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] I kindly suggest that you ensure the Modal popup data is populated with the data from the projects array.

Reply:
Thanks @topeogunleye. Suggestion implemented